app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -168,19 +170,24 @@ class ProfileChooserFragmentTest {
         )
       ).check(matches(isDisplayed()))
       matchStringOnProfileListItem(
-        position = 0,
+        itemPosition = 0,
         targetView = R.id.profile_last_visited,
         stringToMatch = "${context.getString(R.string.profile_last_used)} just now"
       )
+      // Forcing to remove the LiveData Observer.
+      it.onActivity {
+        profileTestHelper.forceToCompleteOperation(data)
```

Comment:
Why does this need to be done again? Ditto elsewhere?

Reply:
We cannot remove the observer before testing the actual thing, so the flow is like
add observer, test, remove observer, else the test will freeze and nothing get observed.